README.md

```diff
@@ -123,6 +123,12 @@ If you want to enable [2-way SSL Client Authentication](https://docs.gitlab.com/
 
 GitLab includes a number of themes, and you can set the default for all users with this variable. See [the included GitLab themes to choose a default](https://github.com/gitlabhq/gitlabhq/blob/master/config/gitlab.yml.example#L79-L85).
 
+    gitlab_extra_settings:
+    - key: "gitlab_rails['foo']"
```

Comment:
A more natural yaml-ish way to do this would be to do something like this:

```yaml
gitlab_extra_settings:
  gitlab_rails:
    - key: ['foo', 'bar']
      value: 'baz'
```

This would prevent some of the possible typos in the `key` field. What do you think?

Reply:
I had a look to your proposal, and I think that could be better. So I pushed an improvement. This is not exactly what you suggested, but I took into account a part of the idea.

Reply:
Ahh, you're right, it looks like the way `gitlab.rb` works, attributes can only have one key depth. I knew this, but I spend too much time reading the Ruby code that implements this magic. :smile: